### PR TITLE
docs(android): formatting and version updates

### PR DESCRIPTION
**Pull request by space-pope**
### PR Checklist

Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [x] I am requesting to **pull a topic/feature/bugfix branch** (right side). In other words, not _master_.
- [x] I have run `npm test` against my changes and tests pass.
- [x] I have added or edited necessary documentation, or no docs changes are needed.

### Description

Updates for Android documentation, mostly some catch-up work to make the various pages fit more naturally with the current state of the library and code block reformatting to make it a little friendlier for the site's text column width.

### CODE REVIEW

**Comment by timmywil** [comment]
@space-pope seems like you're removing some useful info in the cookbook about how to include the default Spokestack wakeword, or is that done automatically if no wakeword is specified?

**Comment by space-pope** [comment]
I got rid of that because the cookbook is more for nonstandard configurations and tips, and wakeword is enabled by default in the new architecture (as long as you tell it where to find the models). The actual word itself that the app listens for is determined by the models themselves rather than any config property; there's no equivalent in Android of the `appleWakeword` iOS component. The default configuration is covered in a couple other places.

**Comment by timmywil** [comment]
@space-pope I just know that it's the only place where the Tensorflow lite Spokestack wakeword model URLs are available as links in the Android docs.

**Comment by space-pope** [comment]
They're [here](https://github.com/spokestack/spokestack.io/blob/develop/content/docs/Android/turnkey-configuration.md#speech-pipeline) too.

**Comment by timmywil** [comment]
Oh right, we haven't deployed that yet.

**Review by timmywil — APPROVED** [review]
LGTM

File: content/docs/Android/cookbook.md [hunk 1]
```diff
@@ -7,83 +7,59 @@ draft: false
...
-    val size = inputStream.available()
-    val buffer = ByteArray(size)
-    inputStream.read(buffer)
-    inputStream.close()
-    val fos = FileOutputStream(filterFile)
-    fos.write(buffer)
-    fos.close()
-}
-```
+Note the use of `useProfile()`. Available profiles can be found [here](https://www.javadoc.io/static/io.spokestack/spokestack-android/%ANDROID_VERSION/io/spokestack/spokestack/profile/package-summary.html)
```
Comment (noelweichbrodt) [inline content/docs/Android/cookbook.md:39]: this url is a `400`

**Comment by noelweichbrodt** [comment]
Is there some sort of markup variable system I'm not aware of where `%ANDROID_VERSION` is defined?

**Review by noelweichbrodt — APPROVED** [review]
Uncontroversial updates

**Comment by noelweichbrodt** [comment]
ah i see, it's a gatsby mechanism

**Comment by space-pope** [comment]
@noelweichbrodt Yeah; it's brand-new (to us). Declare your variables in `gatsby-config`, and they're straight string replacements in the markdown.

**Pull request merged**